src/points/PointsDiscoverCard.tsx

```diff
@@ -36,63 +36,68 @@ export default function PointsDiscoverCard() {
   }
 
   return (
-    <View style={styles.container}>
-      <Touchable
-        style={styles.touchable}
-        borderRadius={Spacing.Smallest8}
-        onPress={handlePress}
-        testID="PointsDiscoverCard"
-      >
-        <>
-          <View style={styles.header}>
-            <Text style={styles.title}>{t('points.discoverCard.title')}</Text>
-            <View style={styles.pill}>
-              <Text style={styles.balance}>{pointsBalance}</Text>
-              <LogoHeart size={Spacing.Regular16} />
-            </View>
-          </View>
+    <Touchable
+      shouldRenderRippleAbove
+      style={styles.touchable}
+      borderRadius={Spacing.Smallest8}
+      onPress={handlePress}
+      testID="PointsDiscoverCard"
+    >
+      <>
+        <Image style={styles.image} source={pointsCardBackground} />
+        <View style={styles.content}>
+          <Text style={styles.title}>{t('points.discoverCard.title')}</Text>
           <Text style={styles.description}>{t('points.discoverCard.description')}</Text>
-        </>
-      </Touchable>
-    </View>
+          <View style={styles.pill}>
```

Comment:
nit: You can remove this outer `View` and merge `styles.pill` and `styles.balance`

Reply:
apparently, not on iPhone 😢:

![image](https://github.com/valora-inc/wallet/assets/2737872/cc6878e4-3a4f-42f5-b859-8456758a2782)

Reply:
Ohhh weird, I tried this on android and it worked fine for me, weird that the behavior is inconsistent!
![2024-07-02-130916_950x505_scrot](https://github.com/valora-inc/wallet/assets/569401/b23ac400-b433-4ffb-8c84-20693a065df3)

Reply:
I found that `borderRadius: 100` is rendering weirdly on iOS, while in Android it does the same as on the Web: any big enough value yields perfectly rounded sides. If I set `borderRadius` somewhere equal to the actual height of the element iOS renders it kind of okay (plus `overflow:hidden`). But I prefer to not bother about keeping border radius in sync with the height, just have a big enough value that always works :)